__tests__/client.test.js

```diff
@@ -1,9 +1,17 @@
 import Enzyme from 'enzyme';
 import React from 'react';
 import Adapter from 'enzyme-adapter-react-16';
+import App from '../src/Components/App';
 
 Enzyme.configure({ adapter: new Adapter() });
 
```

Comment:
Your configuration for Enzyme is currently in the same location as your test file. Consider separating these, or else you have to add the Enzyme configuration at the beginning of each test file.